pkg/scalers/aws_sqs_queue_scaler.go

```diff
@@ -21,14 +21,10 @@ const (
 	targetQueueLengthDefault           = 5
 	activationTargetQueueLengthDefault = 0
 	defaultScaleOnInFlight             = true
+	defaultScaleOnDelayed              = false
 )
 
-var awsSqsQueueMetricNamesForScalingInFlight = []string{
-	"ApproximateNumberOfMessages",
-	"ApproximateNumberOfMessagesNotVisible",
-}
-
-var awsSqsQueueMetricNamesForNotScalingInFlight = []string{
+var awsSqsQueueMetricNamesForScaling = []string{
 	"ApproximateNumberOfMessages",
 }
```

Comment:
Does keeping this here make sense? I mean, maybe we could just move it to the parsing function